@here/harp-mapview/lib/Tile.ts

```diff
@@ -604,6 +604,14 @@ export class Tile implements CachedResource {
             return;
         }
 
+        // creates an empty plane if there are no objects on it, but it must be rendered
+        if (this.m_decodedTile === undefined) {
+            if (this.dataSource.addTileBackground && this.m_forceHasGeometry) {
+                this.addPlane();
```

Comment:
... it looks like you're adding new `plane` (what plane?) every frame ...

Please check it, i may miss some fact.

Reply:
yes, this is an old behavior, I did not modify this logic, only separate this function so that I could render empty tiles.

Reply:
as well as in each frame all geometry is removed from the video memory and then created again in the "createGeometries" method,
I understand it should be refactored but not in the context of this task

Reply:
You changed the logic to the entrance of the function as @zbigg mentioned above. m_decodedTile is always undefined, it is for one frame set, then it is cleared again in.